src/applications/simple-forms/21-0845/pages/organizationReps.js

```diff
@@ -6,11 +6,15 @@ import OrgRepsViewField from '../components/OrgRepsViewField';
 export default {
   uiSchema: {
     'ui:title': (
-      <h3 className="custom-header">Organization’s representatives</h3>
+      <>
+        <p className="vads-u-font-size--lg">Organization’s representatives</p>
```

Comment:
wouldn't we want to keep this in at h3? and just style it how we want?

Reply:
It's the Review page where using <h> tags gets you into a11y-hierarchy issues; not on the form-page itself.  I just found out the hard way.